src/Transport/AzureServiceBusTransportSettingsExtensions.cs

```diff
@@ -11,14 +11,17 @@
     /// </summary>
     public static partial class AzureServiceBusTransportSettingsExtensions
     {
+        const string Note = "As long as the persistence configuration API has not been adjusted to match the transport configuration API keep bumping the versions when working on a new major";
```

Comment:
Is there an issue or documentation we can link to help Future Dev determine if the persistence configuration has been adjusted to match the transport configuration?

Reply:
Unfortunately, at this stage we have only an internal tracking issue that we cannot link in the public documentation :(